flavors/template-Exasol-all-python-3.8-conda/flavor_base/build_deps/packages/apt_get_packages

```diff
@@ -2,7 +2,6 @@ coreutils|8.28-1ubuntu1
 locales|2.27-3ubuntu1.5
```

Comment:
Please remove the whole file, it is not used anymore

Reply:
Done